ios/FluentUI/Extensions/UIView+Extensions.swift

```diff
@@ -6,6 +6,11 @@
 import UIKit
 
 extension UIView {
+
+    var viewScreen: UIScreen {
+        return window?.screen ?? .main
```

Comment:
The UIWindows' screen property is deprecated:
https://developer.apple.com/documentation/uikit/uiwindow/1621597-screen

We shouldn't be using it.
Please remove this computed property.